templates/scripts/add-vpn-user.sh

```diff
@@ -15,8 +17,7 @@ fi
 EXEC="kubectl -n vpn exec -it $POD -- /bin/bash -c"
 
 # get name
-echo -n "Enter your name: "
-read name
+echo -n "Enter your name: " && read name
```

Comment:
Let's print the name of the cluster before this so they know which context they are using. Otherwise they don't really know where they are adding the user to, and I don't think we explicitly tell them to change their kube context.

Reply:
done